src/UI/Core/TextArea.h

```diff
@@ -14,6 +14,7 @@ class TextArea : public Control
 	TextArea() = default;
 	virtual ~TextArea() = default;
 
+	void textColor(int r, int g, int b, int a = 255) { textColor(NAS2D::Color{static_cast<uint8_t>(r), static_cast<uint8_t>(g), static_cast<uint8_t>(b), static_cast<uint8_t>(a)}); };
```

Comment:
Is there a way to remove the use of `static_cast`?

Reply:
No. char/unsigned char/signed char all require down casts.

I wanted to remove it entirely, but that broke a call in `FactoryProduction.cpp`. Decided to leave it as a separate issue.

Reply:
Yes, I was thinking update the call. Is it just one place?

Reply:
As far as the compiler warned me it is.